Automattic/wp-calypso
# DO NOT MERGE: Test FullStory FullStory Analytics Components

#### Changes proposed in this Pull Request

* Example of how to test #50576

#### Testing instructions

**Note: To get good results from the recorded sessions I use Proxyman to point `wordpress.com` to my local Calypso. Otherwise it looks like an un-styled mess**

1. Verify recording is enabled on a8c FullStory account
1. Comment out `client/components/full-story-recorder/index.tsx:41` ( `devMode: 'development' === process.env.NODE_ENV` )  to enable FullStory in dev.
2. Boot branch locally.
3. Navigate to `/backup/:siteSlug` on a Jetpack site
4. Navigate through dates until "Download Backup" is clickable
5. Go through Download flow, until "Your backup is now available for download." is visible.
6. Go to FullStory "Segments" dashboard.
7. Verify session is visible, and "private by default" and "excluding" are woking as expected ( see screenshot below ).
<img width="1311" alt="Screen Shot 2021-05-06 at 1 22 38 PM" src="https://user-images.githubusercontent.com/2810519/117361969-8de30380-ae6f-11eb-9908-f3cadea54c1f.png">



Related to #50576
